sphinx-doc/compiling.rst

```diff
@@ -80,7 +80,7 @@ Set the environment variable SOFTWARE_ROOT to the location you wish to install H
 
 Clone the git repository to get the source::
 
-    $ git clone --recursive https://bitbucket.org/glotzer/hoomd-blue
+    $ git clone --recursive https://github.com/glotzerlab/hoomd-blue
 
 By default, the *maint* branch will be checked out. This branch includes all bug fixes since the last stable release.
```

Comment:
I'm not sure that this is accurate now - I just cloned the repo and I got the `master` branch. May need to change a setting.